public/js/login.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+function setTime() {
```

Comment:
because this combines logic and view, want to name it something that indicates it sets the time in the view ...